mne/data/html_templates.py

```diff
@@ -100,3 +100,32 @@
     </tr>
 </table>
 """)
+
+evoked_template = Template("""
+<table class="table table-hover">
+    <tr>
+        <th>Name</th>
+        <td>{{evoked.comment}}</td>
```

Comment:
Maybe a term like "Content" would fit better here? Check for example what `evoked.comment` looks like when you average epochs from, say, 4 event types and with different counts. I wouldn't want to call the monster you'll find in `evoked.comment` a "name" anymore 😅😱